crates/pathfinder/src/state/sync/l2.rs

```diff
@@ -544,6 +544,7 @@ mod tests {
         const DEF0: &str = r#"{
             "abi": [],
             "program": {
+                compiler_version": "0.10.0",
```

Comment:
I guess this was technically not required? Don't mind it being here, just checking my understanding is correct.

Reply:
No, you're right, this is technically not required. Shall I remove it?

Reply:
Maybe, just to make it less confusing when `git blame` in the future.

Reply:
Removed. Was syntactically incorrect anyway ;)